instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py

```diff
@@ -501,12 +501,12 @@ def _converse_on_success(
 
         # In case of an early stream closure, the result may not contain outputs
         if self._stream_has_output_content(result):
-            event_logger = instrumentor_context.event_logger
+            logger = instrumentor_context.logger
             choice = _Choice.from_converse(result, capture_content)
             # this path is used by streaming apis, in that case we are already out of the span
             # context so need to add the span context manually
             span_ctx = span.get_span_context()
-            event_logger.emit(
+            logger.emit(
                 choice.to_choice_event(
```

Comment:
Here I think we should put the span in context and pass the context instead of the 3 deprecated params.